libraries/rush-lib/src/utilities/test/objectUtilities.test.ts

```diff
@@ -0,0 +1,166 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See the @microsoft/rush package's LICENSE file for license information.
+
+describe('objectUtilities', () => {
+  describe('objectsAreDeepEqual', () => {
+    it('can compare primitives', async () => {
+      const { objectsAreDeepEqual } = await import('../objectUtilities');
```

Comment:
The use of `await import()` in unit tests is really only relevant for code that needs to have mocked dependencies, which this isn't. Making all the tests synchronous will likely improve test performance.